### DISCUSSION TITLE
Tracking Issue

### DISCUSSION THREAD

**Discussion by shapley** [opengisch/QField#4576] [Bugs]
Hi,

A small group of three people went out to try 3 Android 10 or above devices (3 x samsung galaxy's plus one galaxy tablet) and an iOS device and tested them all with 'Tracking' enabled. We found that 2 Android devices were able to plot points every 30 seconds (default settings on all devices were checked to ensure they were the same) whereas the iOS, SG Tablet, and two Android devices would only plot the first track point then the gps arrow 'seeks' in a continuous line but doesn't plot any further points. 

At first I thought it was the QField settings in each device but they were all identical. Has anyone had this issue? I think it is possible there is a setting somewhere to rectify this problem but don't know where to start looking. Any hints to check?

**Comment by timbo-barnes** [comment]
You will get this behaviour if locking the device (using the normal device lock) or switching apps. QField doesn't (yet) have background tracking so as soon as the app is switched out of focus or locked, GPS positions won't be recorded.
The way around this is to use the 'Lock' within the QField either by long-pressing the map or from within the 3-bar menu: This keeps Qfield running and locations being recorded.

**Reply by gis-kmg** [reply]
Any ETA on background tracking feature on iOS please? :) @nirvn
